src/components/Button/styles.module.css

```diff
@@ -0,0 +1,34 @@
+.button {
+    height: 52px;
+    border-radius: 6px;
+    background-color: #EB5B14;
+    color: #FFF;
+    border: 0;
+    padding: 0 120px;
+    font-size: 20px;
```

Comment:
No figma nós utilizamos `16px` para a label desses buttons.
Convertendo isso pra rem, fica `1rem`.
ref: https://nekocalc.com/px-to-rem-converter

Reply:
👍